Skia build fails with `The "RuntimeAssetsSelectorTask_v0" task was not found`

### Current behavior

```
44>C:\Users\PC\Desktop\uno\build\nuget\uno.winui.runtime-replace.targets(50,3): error MSB4036: The "RuntimeAssetsSelectorTask_v0" task was not found. Check the following: 1.) The name of the task in the project file is the same as the name of the task class. 2.) The task class is "public" and implements the Microsoft.Build.Framework.ITask interface. 3.) The task is correctly declared with <UsingTask> in the project file, or in the *.tasks files located in the "C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\amd64" directory.
44>Done building project "UnoIslands.Skia.Wpf.csproj" -- FAILED.
```

This is likely related to https://github.com/unoplatform/uno/pull/14122

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

@Youssef1313, That's was for **SamplesApp.Skia.Wpf** & **UnoIslands.Skia.Wpf**.
But that's quite interesting.

I closed changes from your branch and opened **master** in another solution.

Without your changes I have the following errors stack for **UnoIslands.Skia.Wpf**:

![image](https://github.com/unoplatform/uno/assets/69314237/98d413a0-9cbf-4b8f-a29e-3e96af5aaa24)

```
30>R:\limeniye_uno_latest\build\nuget\uno.winui.runtime-replace.targets(50,3): error MSB4036: The "RuntimeAssetsSelectorTask_v0" task was not found. Check the following: 1.) The name of the task in the project file is the same as the name of the task class. 2.) The task class is "public" and implements the Microsoft.Build.Framework.ITask interface. 3.) The task is correctly declared with <UsingTask> in the project file, or in the *.tasks files located in the "C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\amd64" directory.
30>Done building project "UnoIslands.Skia.Wpf.csproj" -- FAILED.
30>R:\limeniye_uno_latest\build\nuget\uno.winui.runtime-replace.targets(50,3): error MSB4036: The "RuntimeAssetsSelectorTask_v0" task was not found. Check the following: 1.) The name of the task in the project file is the same as the name of the task class. 2.) The task class is "public" and implements the Microsoft.Build.Framework.ITask interface. 3.) The task is correctly declared with <UsingTask> in the project file, or in the *.tasks files located in the "C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\amd64" directory.
30>Done building project "UnoIslands.Skia.Wpf.csproj" -- FAILED.
```

But, after that I closed VS with the master branch and opened solution with your branch again and this errors was disappeared.

--- a/src/SamplesApp/UnoIslands.Skia.Wpf/UnoIslands.Skia.Wpf.csproj
+++ b/src/SamplesApp/UnoIslands.Skia.Wpf/UnoIslands.Skia.Wpf.csproj
@@ -66,6 +66,12 @@
 	  </Compile>
 	</ItemGroup>
 
+	<PropertyGroup>
+		<UnoUIMSBuildTasksPath>$(MSBuildThisFileDirectory)..\..\SourceGenerators\Uno.UI.Tasks\bin\$(Configuration)_Shadow</UnoUIMSBuildTasksPath>
+	</PropertyGroup>
+
+	<Import Project="..\..\SourceGenerators\Uno.UI.Tasks\Content\Uno.UI.Tasks.targets" Condition="'$(SkipUnoResourceGeneration)' == '' " />
+
 	<Import Project="..\..\..\build\nuget\*.Skia.Wpf.props" />
 	<Import Project="..\..\..\build\nuget\*.Skia.Wpf.targets" />
 	<Import Project="..\..\..\build\nuget\uno.winui.runtime-replace.targets" />

